contracts/ERC20TimedMint.sol

```diff
@@ -48,6 +46,14 @@ contract ERC20TimedMint is ERC20 {
         require(amount <= mintCap, "ERC20: Mint exceeds maximum amount");
         super._mint(to, amount);
     }
+
+    function _setTimeUntilNextMint(uint256 _timeDelay) external {
```

Comment:
Honestly, just set this and `_setMintCap()` to internal and it will be called on the child contract.